### ISSUE TITLE
[Request]: FSR 2 / FSR 3.1 (unreleased) support

### ISSUE THREAD

**Issue by LuvbCa** [xCollateral/VulkanMod#387]
Heyyo,

i was wondering wheter the code base is compatible with the fsr 2 / fsr 3.1 pipeline because im interested in implementing this feature to this mod. This could really benefit users of shaders.

anyways thanks in advance :)

**Comment by ajdgsu** [comment]
Yeah also I noticed that AMD FSR 3.1 officially supported vulkan api just weeks ago. (https://www.phoronix.com/news/AMD-FSR-3.1-Announced) So looks pretty promising.

**Comment by DesolateIntention** [comment]
This would be insanely epic.

**Comment by shellbamboo** [comment]
Very cool

**Comment by addre98** [comment]
it will be so cooooool！
